Expecto/Expecto.fs

```diff
@@ -454,7 +454,18 @@ module Impl =
       | Passed -> "Passed"
       | Ignored reason -> "Ignored: " + reason
       | Failed error -> "Failed: " + error
-      | Error e -> "Exception: " + e.ToString()
+      | Error e -> 
+        let rec printInner (ex: exn) msg =
```

Comment:
Wouldn't it be better to have this outside of the `TestResult` type to keep concerns separate?

Reply:
Agree, I move it to an `Impl` module and mark as a private.